### DISCUSSION TITLE
Parameterisation - Define each cell component individually

### DISCUSSION THREAD

**Discussion by Sunnyturicum** [pybamm-team/PyBaMM#3255] [Q&A] [answered]
I watched [this](https://www.youtube.com/watch?v=oUhOL8ebOFw) YouTube tutorial about parametrisation. However, in the latest PyBaMM version, the structure of parameters is organised in parameter sets only, without having a modular system for each cell component. Is there a way to define each cell component individually?

Here an example of what I am trying to do:
```
parameter_values = pybamm.ParameterValues(
    chemistry={
        "chemistry": "lithium_ion",
        "cell": "OKane2022",
        "negative electrode": "OKane2022",
        "separator": "OKane2022",
        "positive electrode": "Prada2013",
        "electrolyte": "OKane2022",
        "experiment": "OKane2022",
    }
)
```

**Comment by brosaplanella** [comment, marked as answer]
You can write a parameter script (like the ones in the current approach) that imports various parameters from other files.
